documents/postman/staging-simplified.postman_environment.json

```diff
@@ -0,0 +1,66 @@
+{
+	"id": "1acffa98-51c5-4ba7-87fc-9f3d460c38ba",
+	"name": "staging-vanotify",
+	"values": [
+		{
+			"key": "email-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "notification-api-url",
+			"value": "https://staging-api.va.gov/vanotify",
+			"enabled": true
+		},
+		{
+			"key": "notification-api-url-private",
+			"value": "https://staging.api.notifications.va.gov/vanotify",
+			"enabled": true
+		},
+		{
+			"key": "notification-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-api-key",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "service-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "sms-sender-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "sms-template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "template-id",
+			"value": "",
+			"enabled": true
+		},
+		{
+			"key": "api-key-id",
+			"value": "",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "jwt_signed",
+			"value": "",
+			"type": "any",
+			"enabled": true
+		}
```

Comment:
@edDocMe360 I noticed in the push notification body there's 2 variables that we don't have listed here:
![image](https://github.com/department-of-veterans-affairs/notification-api/assets/99204969/935ee5d7-d027-4cf4-9bd1-6072adbae521)

Don't want to start changing the body example in the collection for this ticket, so can we just add those variables here?

Reply:
Added. These weren't in the original collections but we should have them in there. Good catch. Added them to the original collections as well.